components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2980,6 +2980,22 @@ components:
         defaultValue:
           type: string
           example: 'Option 1'
+    GrantTypeMetaData:
+      type: object
+      properties:
+        options:
+          type: array
+          items:
+            $ref: '#/components/schemas/GrantTypes'
+    GrantTypes:
```

Comment:
shall we make this GrantType?

Reply:
Fixed in bced918352ff5a126a771d5a00ad510b905e8597